osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -583,6 +574,70 @@ protected override void Dispose(bool isDisposing)
             hitEvents.Clear();
         }
 
+        #region Static helper methods
+
+        /// <summary>
+        /// Given an accuracy (0..1), return the correct <see cref="ScoreRank"/>.
+        /// </summary>
+        public static ScoreRank RankFromAccuracy(double accuracy)
+        {
+            switch (accuracy)
+            {
+                case accuracy_cutoff_x:
+                    return ScoreRank.X;
+
+                case >= accuracy_cutoff_s:
```

Comment:
@peppy thought we weren't using these operators near case labels? Not that I'm bothered, just curious.

Reply:
Hmm, not sure. This case seemed pretty legible to me (rider did it) so I went with it. Will change to `if` statements as it reads just as well and doesn't break convention.